x-pack/plugins/ingest_manager/public/applications/ingest_manager/sections/epm/hooks/use_links.tsx

```diff
@@ -0,0 +1,92 @@
+/*
```

Comment:
Hopefully this can be replaced by our generic [useLink](https://github.com/elastic/kibana/blob/feature-ingest/x-pack/plugins/ingest_manager/public/applications/ingest_manager/hooks/use_link.ts) hook in combination with [constants](https://github.com/elastic/kibana/blob/feature-ingest/x-pack/plugins/ingest_manager/public/applications/ingest_manager/constants/index.ts) for app paths

But I am not familiar with all the functionality of this hook, so am totally open to EPM having it's own hooks (either standalone, or wrappers around our generic hooks) if you think it makes sense